gribscan/tools.py

```diff
@@ -20,8 +28,14 @@ def create_index():
     )
     args = parser.parse_args()
 
+    if args.outdir is not None:
+        indices = [Path(args.outdir) / (Path(s).stem + ".index") for s in args.sources]
```

Comment:
It seems like a very similar functionality is implemented inside [`write_index`](https://github.com/gribscan/gribscan/blob/37ec8350c94bd5b925869fd34652bbc8cd368892/gribscan/gribscan.py#L320C5-L320C5) already, which to me raises the question if passing `args.outdir` into `write_index` as an additional keyword argument would lead to a cleaner implementation.

Reply:
Actually this is what happens. However, `write_index` requires an actual path to the index file not only an output directory. That's why I create them here to pass to `write_index`.

Reply:
Yes, but `write_index` could get another optional argument, couldn't it?

Reply:
Ahh, sorry I misread your first comment. Yes, this might turn out cleaner, I will give it a try.